packages/react-native-renderer/src/ReactFabricHostConfig.js

```diff
@@ -293,6 +293,8 @@ export function prepareUpdate(
   );
   // TODO: If the event handlers have changed, we need to update the current props
   // in the commit phase but there is no host config hook to do it yet.
+  // So instead we hack it by updating it in the render phase.
+  instance.canonical.currentProps = newProps;
```

Comment:
So this is sufficient to fix the crashes? How important is the follow-up PR to add a commit phase hook?

Reply:
It doesn't matter for sync mode but will be important eventually.

I'm considering using the memoizedProps solution to avoid having to add a commit phase in JS. Currently the commit phase is entire in native.